java/languages.antlr/src/org/netbeans/modules/languages/antlr/v4/Antlr4CompletionProvider.java

```diff
@@ -143,12 +129,12 @@ protected void query(CompletionResultSet resultSet, Document doc, int caretOffse
                         tokens.previous();
                         lookAround(fo, tokens, caretOffset, prefix, resultSet);
                     } else {
-                        //Empty grammar so far offer lexer and grammar
-                        addTokens("", caretOffset, resultSet, "lexer", "grammar");
+                        //Empty grammar so far offer lexer, parser and grammar
+                        addTokens("", caretOffset, resultSet, "lexer", "parser", "grammar");
                     }
                 }
             } catch (Throwable th) {
-                System.out.println(th);
+                //System.out.println(th);
```

Comment:
was this intended?  Would be better if it wouldn't be `Throwable`, It is always a bit dangerous to catch that without handling the edge cases.

Reply:
Well, removed that one.